value_test.go

```diff
@@ -817,3 +817,39 @@ func TestValueArrayBufferContents(t *testing.T) {
 		t.Fatalf("Expected an error trying call SharedArrayBufferGetContents on value of incorrect type")
 	}
 }
+
+func TestValueStrictEquals(t *testing.T) {
+	ctx := v8.NewContext()
+	defer ctx.Close()
+
+	numberOne, err1 := ctx.RunScript("1", "")
+	numberOneB, err2 := ctx.RunScript("1", "")
+	numberTwo, err3 := ctx.RunScript("2", "")
+	stringOne, err4 := ctx.RunScript("'1'", "")
+	function, err5 := ctx.RunScript("const fn = () => {}; fn", "")
+	sameFunction, err6 := ctx.RunScript("fn", "")
+	anotherFunction, err7 := ctx.RunScript("const fn2 = () => {}; fn2", "")
+
+	if err := errorsJoin(err1, err2, err3, err4, err5, err6, err7); err != nil {
+		t.Fatal("Error getting test values", err)
+	}
+
+	if !numberOne.StrictEquals(numberOneB) {
+		t.Fatalf("Number 1 and Number 1 should be strict equal")
```

Comment:
Since the following assertions are independent, using `t.Errorf` is appropriate.